File: frontEnd/src/app/component/models/LaporanItem.ts
import { Guid } from "guid-typescript";
import { User } from "./user";

export class LaporanItem {
    laporanItemId: Guid = Guid.createEmpty();
    laporanHeaderId: Guid = Guid.createEmpty();
    keterangan: string = "";
    tglBuat: Date = null;
    userId: Guid = Guid.createEmpty();
    isDeteled: boolean = false;
    user: User = new User;
}

File: backEnd/Models/LaporanItem.cs
using System;

namespace backEnd.Models
{
    public class LaporanItem
    {
        public Guid LaporanItemId { get; set; }
        public Guid LaporanHeaderId { get; set; }
        public string Keterangan { get; set; }
        public DateTime TglBuat { get; set; }
        public Guid UserId {get; set;}
        public bool IsDeteled { get; set; }
        public User User {get; set;}
    }
}